packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -855,12 +855,10 @@ class ScrollableBaseNative extends Component {
 
 	scrollTo = (opt) => {
 		if (!this.deferScrollTo) {
-			const
-				{left, top} = this.getPositionForScrollTo(opt),
-				{scrollTo} = this.props;
+			const {left, top} = this.getPositionForScrollTo(opt);
```

Comment:
Do we need to use `this.props.scrollTo` instead of `props.scrollTo`?

Reply:
The previous way, the context was broken so I had to change this way.